src/Authentication/Authentication/Constants.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Graph.PowerShell.Authentication
     using System.IO;
     public static class Constants
     {
-        public const string SDKHeaderValue = "Graph-powershell-{0}-{1}.{2}.{3}";
+        public const string SDKHeaderValue = "Graph-powershell-{0}.{1}.{2}";
```

Comment:
any chance anyone has a dependency on the format?  I can't think of any dashboard that should break based on the format but thought i'd ask.

Reply:
This shouldn't break anyone since the SDKHeader was not being logged to the debug stream in the old implementation. 

The only tool that takes a dependency on the SDKHeader is the Graph telemetry dashboard. This fix address the formatting issue that currently exists with the dashboard since it expects the header match the spec - https://github.com/microsoftgraph/msgraph-sdk-design/blob/master/middleware/TelemetryHandler.md.